tools/secret-management/Azure.Sdk.Tools.SecretRotation.Configuration/RotationConfiguration.cs

```diff
@@ -1,6 +1,7 @@
 using System.Collections.ObjectModel;
 using Azure.Sdk.Tools.SecretRotation.Core;
 using Microsoft.Extensions.Logging;
+using TimeProvider = Azure.Sdk.Tools.SecretRotation.Core.TimeProvider;
```

Comment:
Any reason not to just delete https://github.com/Azure/azure-sdk-tools/blob/main/tools/secret-management/Azure.Sdk.Tools.SecretRotation.Core/TimeProvider.cs and use https://learn.microsoft.com/en-us/dotnet/api/system.timeprovider?view=net-8.0 now?

Reply:
I can but while upgrading the 40+ projects I didn't refactor anything outside of necessary changes for the net6->net8 upgrade. I have the time now so I can do this. I'll push another commit.